packages/graphql/src/utils/hasProxy.js

```diff
@@ -0,0 +1,38 @@
+import contracts from '../contracts'
+
+export default async function hasProxy(address) {
```

Comment:
nit: add a bit of comments to explain this magic :)

Reply:
Will do